crates/sui-config/src/genesis.rs

```diff
@@ -706,6 +706,7 @@ fn create_genesis_checkpoint(
         previous_digest: None,
         epoch_rolling_gas_cost_summary: Default::default(),
         next_epoch_committee: None,
+        timestamp_ms: 0, /* todo - put actual ceremony timestamp */
```

Comment:
probably makes sense to just call 'now' and stick in the current timestamp

Reply:
Is it guaranteed that this invoked once for the genesis? Eg people are not running this on different machines or something?

Reply:
oh wait...hmmm there may be 1 potential issue. nvm ignore this comment for now, i'll revisit this later.

Reply:
Ok, rolled back this change

Reply:
Yep, thats what I did